lib/java-wrapper.ts

```diff
@@ -76,24 +78,28 @@ export async function getCallGraph(
   targetPath: string,
   timeout?: number,
 ): Promise<Graph> {
-  const jarPath = await fetch(
-    config.CALL_GRAPH_GENERATOR_URL,
-    config.CALL_GRAPH_GENERATOR_CHECKSUM,
-  );
-  const targets = await timeIt('getEntrypoints', () => getTargets(targetPath));
+  const [jarPath, targets, { tmpDir, classPathFile }] = await Promise.all([
+    await fetch(
```

Comment:
I think you don't need the `await` inside `Promise.all`

Reply:
You are of course right :-) It shouldn't do any harm, but I will remove it. It's a copy&paste leftover.